main.js

```diff
@@ -0,0 +1,83 @@
+//bai1
+const getNumber = n => {
+    n == Number
+    for (i = 1; i <= 10; i++) {
+        console.log('${n} * ${i} =', i)
+    }
+}
+
+//bai2
+const takeEvenNumber = n => {
+    if (n >= 1 && n <= 30) {
+        for(let i =2; i <= n; i += 2)
+        console.log(i)
+    }
+}
+
+//bai3
+const calcSum = number => {
+    let sum = 0
+    if (n >= 1 && n <= 30) {
+        for(let i = 1; i <= n; i++) {
+            sum += i
+        }
+        console.log(sum)
+    }
+}
+
+//bai4
+const takeFactorial = n => {
+  n == Number
+  let fact = 1
+  if (n >= 1 && n <=30) {
+    for(let i = 1; i <= n; i++) {
+      fact *= i
+    }
+    console.log(fact)
+  }
+}
+
+//bai5
+let arr = [1, 2, 3, 4, 5, 6]
```

Comment:
`arr` không bị thay đổi (gán lại) bên dưới nên hãy khai báo là `const`
```suggestion
const arr = [1, 2, 3, 4, 5, 6]
```